com/win32comext/axdebug/codecontainer.py

```diff
@@ -17,13 +18,13 @@
 from win32com.axdebug.util import _wrap
 from win32com.server.exception import COMException
 
-_keywords = {}  # set of Python keywords
-for name in """
- and assert break class continue def del elif else except exec
- finally for from global if import in is lambda not
- or pass print raise return try while
- """.split():
-    _keywords[name] = 1
+_keywords = {
+    _keyword
+    for _keyword in kwlist
+    # Avoids including True/False/None
+    if _keyword.islower()
+}
+"""set of Python keywords"""
```

Comment:
This is also done in https://github.com/mhammond/pywin32/pull/2439 for easier review since it changes behaviour.